tests/Dockerfile

```diff
@@ -27,4 +27,23 @@ RUN apt-get update && \
 RUN pip install --upgrade pip && \
     pip install tox
 
-CMD [ "tox" ]
+RUN apt-get -y --no-install-recommends install \
+    ca-certificates \
+    curl
+
+RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
+RUN curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.4/gosu-$(dpkg --print-architecture)" \
+    && curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.4/gosu-$(dpkg --print-architecture).asc" \
+    && gpg --verify /usr/local/bin/gosu.asc \
+    && rm /usr/local/bin/gosu.asc \
+    && chmod +x /usr/local/bin/gosu
+
+COPY entrypoint.sh /usr/local/bin/entrypoint.sh
+
+WORKDIR /shared
+# Put control_node_serial_interface binary in the system path
+ENV PATH="/shared/control_node_serial:${PATH}"
```

Comment:
is that really needed ?

Reply:
Good catch, this is a leftover from previous debugging tests I made. It's not required.